commands/operator-sdk/cmd/build.go

```diff
@@ -167,6 +170,27 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 
 	log.Infof("Building Docker image %s", baseImageName)
 
+	dbArgs := []string{"build", ".", "-f", "build/Dockerfile", "-t", baseImageName}
+
+	if dockerBuildArgs != "" {
+		splitArgs := strings.Fields(dockerBuildArgs)
+		dbcmd := exec.Command("docker", append(dbArgs, splitArgs...)...)
+		if err := projutil.ExecCmd(dbcmd); err != nil {
+			if enableTests {
+				return fmt.Errorf("failed to output intermediate image %s: (%v)", image, err)
+			}
+			return fmt.Errorf("failed to output build image %s: (%v)", image, err)
+		}
+	} else {
+		dbcmd := exec.Command("docker", dbArgs...)
+		if err := projutil.ExecCmd(dbcmd); err != nil {
+			if enableTests {
+				return fmt.Errorf("failed to output intermediate image %s: (%v)", image, err)
+			}
+			return fmt.Errorf("failed to output build image %s: (%v)", image, err)
+		}
+	}
+
```

Comment:
There's a decent amount of repeated code here. Can we calculate the extra build args (which may end up being empty), and just append them to dbArgs?

Same comment for the test image below as well.

Reply:
Now configured to append additional args (if they exist) to the base package of args and then run build command against the full suite of args. The exec code now only appears once after additional args may have been added.